setup.py

```diff
@@ -99,7 +99,8 @@
     'SeisHub', 'Seismic Handler', 'seismology', 'seismogram', 'seismograms',
     'shapefile', 'signal', 'slink', 'spectrogram', 'StationXML', 'taper',
     'taup', 'travel time', 'trigger', 'VERCE', 'WAV', 'waveform', 'WaveServer',
-    'WaveServerV', 'WebDC', 'web service', 'Winston', 'XML-SEED', 'XSEED', ]
+    'WaveServerV', 'WebDC', 'web service', 'WIN', 'Winston', 'XML-SEED',
+    'XSEED', ]
```

Comment:
'XSEED']

Reply:
it was in your original file :-) but OK, will do :)